---a/net/minecraft/network/play/server/S25PacketBlockBreakAnim.java
+++b/net/minecraft/network/play/server/S25PacketBlockBreakAnim.java
@@ -1,51 +1,47 @@
 package net.minecraft.network.play.server;
 
 import java.io.IOException;
+
 import net.minecraft.network.INetHandler;
 import net.minecraft.network.Packet;
 import net.minecraft.network.PacketBuffer;
 import net.minecraft.network.play.INetHandlerPlayClient;
 import net.minecraft.util.BlockPos;
-import net.minecraftforge.fml.relauncher.Side;
-import net.minecraftforge.fml.relauncher.SideOnly;
 
-public class S25PacketBlockBreakAnim implements Packet
-{
+
+public class S25PacketBlockBreakAnim implements Packet {
+
     private int breakerId;
     private BlockPos position;
     private int progress;
     private static final String __OBFID = "CL_00001284";
 
-    public S25PacketBlockBreakAnim() {}
+    public S25PacketBlockBreakAnim() {
+    }
 
-    public S25PacketBlockBreakAnim(int breakerId, BlockPos pos, int progress)
-    {
+    public S25PacketBlockBreakAnim(int breakerId, BlockPos pos, int progress) {
         this.breakerId = breakerId;
         this.position = pos;
         this.progress = progress;
     }
 
-    public void readPacketData(PacketBuffer buf) throws IOException
-    {
+    public void readPacketData(PacketBuffer buf) throws IOException {
         this.breakerId = buf.readVarIntFromBuffer();
         this.position = buf.readBlockPos();
         this.progress = buf.readUnsignedByte();
     }
 
-    public void writePacketData(PacketBuffer buf) throws IOException
-    {
+    public void writePacketData(PacketBuffer buf) throws IOException {
         buf.writeVarIntToBuffer(this.breakerId);
         buf.writeBlockPos(this.position);
         buf.writeByte(this.progress);
     }
 
-    public void handle(INetHandlerPlayClient handler)
-    {
+    public void handle(INetHandlerPlayClient handler) {
         handler.handleBlockBreakAnim(this);
     }
 
-    @SideOnly(Side.CLIENT)
-    public int func_148845_c()
-    {
-        return this.breakerId;
+    public void processPacket(INetHandler handler) {
+        this.handle((INetHandlerPlayClient) handler);
     }
-
-    public void processPacket(INetHandler handler)
-    {
-        this.handle((INetHandlerPlayClient)handler);
-    }
-
-    @SideOnly(Side.CLIENT)
-    public BlockPos func_179821_b()
-    {
-        return this.position;
-    }
-
-    @SideOnly(Side.CLIENT)
-    public int func_148846_g()
-    {
-        return this.progress;
-    }
-}
+}
